pkg/storer/internal/pinning/pinning.go

```diff
@@ -158,12 +158,16 @@ func (p pinChunkItem) String() string {
 
 // NewCollection returns a putter wrapped around the passed storage.
 // The putter will add the chunk to Chunk store if it doesnt exists within this collection.
-// It will create a new UUID for the collection which can be used to iterate on all the chunks
 // that are part of this collection. The root pin is only updated on successful close of this
 // Putter.
-func NewCollection(st internal.Storage) internal.PutterCloserWithReference {
+// If no UUID is specified in the options, a new UUID will be generated for this collection.
+func NewCollection(st internal.Storage, opts *swarm.CollectionOptions) internal.PutterCloserWithReference {
```

Comment:
The UUID is completely internal to the pinning store. We dont expose it anywhere. In case of export, then import, it is not necessary to reuse the UUID. The UUID is just a unique identifier that ties the root collection Addr to all the chunks it has. 

So remove all these changes related to adding these options. Anyone outside the pinning store doesnt know about this UUID.